.github/workflows/reusable-e2e.yml

```diff
@@ -510,7 +510,6 @@ jobs:
         run: yarn react-core build
 
       - name: Build react-core-notifications package
-        if: steps.restore-package-cache.outputs.cache-hit != 'true'
```

Comment:
There is no cache to draw from, since RN e2e tests run on macos machines, previous steps run on ubuntu